presentation/common/src/presentation-common/content/Fields.ts

```diff
@@ -187,6 +187,16 @@ export class Field {
     });
   }
 
+  public static fromCompressedFieldJSON(json: FieldJSON<string>, classesMap: { [id: string]: CompressedClassInfoJSON }): FieldJSON | undefined {

```

Comment:
The function is already under `Field` class - no need to repeat that in function name. Also, missing comment.